Observer/Base.php

```diff
@@ -42,6 +42,12 @@ public function execute(\Magento\Framework\Event\Observer $observer)
         } catch (\Exception $e) {
             // We should never blow up a customer's site due to bugs in our code.
             $this->logger->critical($e);
+
+            if (getenv('DRIP_RAISE_EXCEPTIONS') == "true") {
```

Comment:
A new feature so that tests explode more vigorously.

Reply:
Prefer `===` when possible. Fetching environment variables directly is kind of an anti-pattern, but I'm not sure what magento lets you do instead.

Reply:
Ach, good catch. I know that about php equality comparison, but I keep forgetting since I don't do it enough. Fixed.